### DISCUSSION TITLE
Welcome to csv2bufr Discussions!

### DISCUSSION THREAD

**Discussion by david-i-berry** [World-Meteorological-Organization/csv2bufr#48] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ammccay** [comment]
Hi, I'm a meteorologist with ECCC. I need help with the mapping json. For some reason, our test files are continually appending the header info between the data points.
Also, not sure how to make the repeating work with csv2bufr, as we only need three descriptors to repeat (007006, 011001 and 011002).

[bufr_map_v4-test.json.zip](https://github.com/wmo-im/csv2bufr/files/11803916/bufr_map_v4-test.json.zip)

Thanks!
Aaron
[few_lines.bufr.txt](https://github.com/wmo-im/csv2bufr/files/11804035/few_lines.bufr.txt)
